### PR TITLE
Card Protocol on xdai now uses DAI.CPXD and CARD.CPXD

### PR DESCRIPTION
**Pull request by lukemelia**
- Sokol still uses DAI and CARD
- This PR updates tests to use DAI.CPXD and CARD.CPXD

### CODE REVIEW

**Comment by backspace** [comment]
Going through the workflows, withdrawal seems broken:

image

will continue going through them, not sure if Michael’s additions might address this, will complete review when done the workflows

**Comment by backspace** [comment]
Should LayerTwoConnectCard be showing the suffixes when I’m on Sokol?

image

**Comment by backspace** [comment]
Issuance has a similar error to withdrawal:

image

**Comment by Aierie** [comment]
> Going through the workflows, withdrawal seems broken:
> 
> image
> 
> will continue going through them, not sure if Michael’s additions might address this, will complete review when done the workflows

This is the bug that Hassan mentioned during standup today that he's working on

File: packages/web-client/app/utils/web3-strategies/types.ts [hunk 1]
```diff
@@ -149,7 +150,7 @@ export interface Layer2Web3Strategy
     options: TransactionOptions
   ): Promise<MerchantSafe>;
   resumeRegisterMerchantTransaction(txnHash: string): Promise<MerchantSafe>;
-  defaultTokenSymbol: ConvertibleSymbol;
+  defaultTokenSymbol: TokenSymbol;
```
[the remark this answers is not in the document]
Comment (Aierie) [reply packages/web-client/app/utils/web3-strategies/types.ts]: Addressed in #2310

File: packages/web-client/tests/acceptance/create-card-space-test.ts [hunk 2]
```diff
@@ -63,7 +63,7 @@ module('Acceptance | create card space', function (hooks) {
     layer2Service.test__simulateRemoteAccountSafes(layer2AccountAddress, [
       createDepotSafe({
         owners: [layer2AccountAddress],
-        tokens: [createSafeToken('DAI', '0')],
+        tokens: [createSafeToken('DAI.CPXD', '0')],
```
[the remark this answers is not in the document]
Comment (lukemelia) [reply packages/web-client/tests/acceptance/create-card-space-test.ts:66]: Done.

**Review by Aierie — APPROVED** [review]

**Pull request merged**